Swiftfin/Views/LibraryView.swift

```diff
@@ -68,13 +71,15 @@ struct LibraryView: View {
         .navigationTitle(viewModel.parent?.displayTitle ?? "")
         .navigationBarTitleDisplayMode(.inline)
         .navBarDrawer {
```

Comment:
If you look at how the navigation bar drawer was implemented, the drawer is shown when this modifier is used no matter what the view passed in is. Instead, we need to use the `.if` modifier and if there are buttons, show the drawer.

This is done on my branch for reference.

Reply:
Ah! Gotcha, okay that makes a lot of sense. I have implemented this the way that you were doing it.